src/ContaoFramework.php

```diff
@@ -81,118 +90,62 @@ class InitializeSystemListener extends AbstractScopeAwareListener
     /**
      * Constructor.
      *
+     * @param ContainerInterface        $container     The dependency injection container
      * @param RouterInterface           $router        The router service
      * @param SessionInterface          $session       The session service
      * @param string                    $rootDir       The kernel root directory
      * @param CsrfTokenManagerInterface $tokenManager  The token manager service
      * @param string                    $csrfTokenName The name of the token
      * @param ConfigAdapter             $config        The config adapter object
+     * @param int                       $errorLevel    The PHP error level
      */
     public function __construct(
+        ContainerInterface $container,
         RouterInterface $router,
         SessionInterface $session,
         $rootDir,
         CsrfTokenManagerInterface $tokenManager,
         $csrfTokenName,
```

Comment:
We don't need to inject `$csrfTokenName `. We can get it from the container, can't we?